index.html

```diff
@@ -1,183 +1,228 @@
+  <meta name="keywords"
+    content="covid, covid resources, covid19, bed, beds, oxygen, ICU, ventilator, covid19, coronavirus, corona, india, virus, carona, karona, korona, covid beds india, covid beds in india, corona beds in india, covid beds delhi, covid beds bangalore, covid beds availability, oxygen supply, covid oxygen, covid resources, covid beds map, covid resources map">
+  <meta property="og:type" content="website">
+  <meta property="og:url" content="https://cowinmap.com">
+  <meta property="og:title" content="The Cowin Map: Verified COVID resources">
+  <meta property="og:description"
+    content="#verified #covid resources from volunteers across #India on Google maps for quick, updated and easy access. Let’s help India win against covid!">
+
+
+  <!-- Google Tag Manager -->
+  <script>(function (w, d, s, l, i) {
+      w[l] = w[l] || []; w[l].push({
+        'gtm.start':
+          new Date().getTime(), event: 'gtm.js'
+      }); var f = d.getElementsByTagName(s)[0],
+        j = d.createElement(s), dl = l != 'dataLayer' ? '&l=' + l : ''; j.async = true; j.src =
+          'https://www.googletagmanager.com/gtm.js?id=' + i + dl; f.parentNode.insertBefore(j, f);
+    })(window, document, 'script', 'dataLayer', 'GTM-5KDBBZ6');</script>
+  <!-- End Google Tag Manager -->
+
+  <script src="https://kit.fontawesome.com/d14f08e7fd.js" crossorigin="anonymous"></script>
+
+
+
+
+
+  <link rel="stylesheet" href="assets/css/style.css">
+  <link rel="stylesheet" href="assets/css/modal.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css">
+</head>
+
+<body class="">
+  <!-- Google Tag Manager (noscript) -->
+  <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-5KDBBZ6" height="0" width="0"
+      style="display:none;visibility:hidden"></iframe></noscript>
+  <!-- End Google Tag Manager (noscript) -->
+  <header>
+    <div class="container">
+      <div class="content">
+        <span class="is-hidden-mobile">
+          <!-- Shown on all screen sizes except mobile -->
+          <h3 class="title is-3">
+            <a href="https://cowinmap.com">The Cowin Map</a>
+          </h3>
+          <p>#verified #covid resources from volunteers across #India on Google maps for quick, updated and easy access.
+          </p>
+          <p>Let’s help India win against covid!</p>
+        </span>
+
+        <span class="is-hidden-tablet">
+          <!-- Shown only on mobile -->
+          <h6 class="title is-5">
+            <a href="https://cowinmap.com">The Cowin Map</a>
+          </h6>
+          <p class="is-size-7">#verified #covid resources from volunteers across #India on Google maps for quick,
+            updated and easy access.</p>
+          <p class="is-size-7">Let’s help India win against covid!</p>
+        </span>
+      </div>
+    </div>
+  </header>
+
+  <div class="modal-wrap" id="modal-demo">
```

Comment:
Could we use the modal from Bulma CSS framework? we already included Bulma CSS  on to this page. So, lets use that. See: https://bulma.io/documentation/components/modal/

Note: Bulma is a CSS framework. So, we will still need the Javascript code written to open and close the Modal.